build.gradle

```diff
@@ -18,7 +18,7 @@
 
 plugins {
     id 'nebula.netflixoss' version '8.9.0'
-    id 'org.jetbrains.kotlin.jvm' version '1.3.70' apply false
+    id 'org.jetbrains.kotlin.jvm' version '1.4.21' apply false
```

Comment:
[Kotlin 1.4.30](https://github.com/JetBrains/kotlin/releases/tag/v1.4.30) came out today.

Reply:
`1.4.+` ? instead of fixing the _patch version_ why not use `+` and leave it to the locks to handle consistency?